tests/test.ts

```diff
@@ -447,3 +448,112 @@ describe('Test clasp logout function', () => {
     expect(dotExists).to.equal(false);
   });
 });
+
+describe('Test clasp help function (in general)', () => {
+  it('should show help correctly', () => {
+    const result = spawnSync(
+      CLASP, ['help'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain(CLASP_USAGE);
+    expect(result.status).to.equal(1);
+  });
+  it('should show help correctly', () => {
+    const result = spawnSync(
+      CLASP, ['--help'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain(CLASP_USAGE);
+    expect(result.status).to.equal(0);
+  });
+  it('should show help correctly', () => {
+    const result = spawnSync(
+      CLASP, ['-h'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain(CLASP_USAGE);
+    expect(result.status).to.equal(0);
+  });
```

Comment:
Prove different descriptions for these cases.
I know it's tedious to write these tests, but they only need to be written correctly once.